addons/l10n_ee/data/account_tax_report_data.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="tax_report_vat" model="account.report">
+    <record id="tax_report" model="account.report">
```

Comment:
This will remove the report and create a new one. Is this really what we want? If not then the xmlid should be renamed in `pre-` in an upgrade script.

Reply:
Done!

Reply:
Not really ;)

I was referring to the change done in the previous versions. For example in 17 here https://github.com/odoo/odoo/commit/f1dc65e46e03f23b131e99d660bb89ffac39bf20#diff-daa0d2840336ccf06e234bcb6c237e5d0a1406dd3c8b8fb0b9ef9e68f80d1e9aR3

If a client upgrades the module locally, still in 17, the report will be dropped and recreated. Thus the upgrade script should be added _here_, alongside the `end-` one you already have.

Anyway, you could leave this PR as is and add the missing script in a new PR from 17.0.